### ISSUE TITLE
Ban passphrases

### ISSUE THREAD

**Issue by Megami-Studios** [aMytho/GlimBoi#106]
Okay, this is a thought that I've had because of some stuff that I've gone through as of late.  Can the banned words list be modified to ban phrases as well?  I'm sure no one wants to ban January, 1, or 2009, but if someone was trying to doxx someone else during a stream (like giving out one's birthday of "January 1, 2009", for example, having the date as a phrase would be a great way to prevent doxxing and autoban a potential stalker, bully.

I don't know how feasible it is (assuming it is at all), but for those dealing with issues, it might help a lot.

**Comment by aMytho** [comment]
Its definitely possible, would require some work to the current system. The current system (word scan) is meant to stop the low effort trolls. If you have someone who is really determined I'm sure they can bypass whatever filter you setup/I create. There are so many ways to spell/communicate words that a filter generally isn't enough.

My current focus with Glimboi is on the companion webapp (basically a streamdeck for the bot). I can try to add moderation phrase filters during that update or after.

**Comment by Megami-Studios** [comment]
True, in the end it's whack a mole, and like I said on the Glimesh thread, better moderation is ultimately the answer, but having additional tools to deal with issues doesn't hurt, either.  Thanks.
